### PR TITLE
docs: add fluentbit/disable audit logs

### PR DESCRIPTION
**Pull request by juliangieseke**
ℹ️ based on this docathon gdoc: https://docs.google.com/document/d/188KEJEI1BdHFxOjlLt2rNJs_lIE6Sgl0MW1kb9dagtA/edit#

@russurquhart1 @cmastr I wasnt sure where to put the fluentbit description, it also could have lived in the tutorial. Is documenting addons something we'd like to do? or shall I move it?

## Jira Ticket
Before creating this pull request, please make sure you have a separate ticket for the docs team to track their work to review and merge, even for minor edits. If you need assistance, ask in the #doc-team channel on slack.

## Checklist
- [ ] Change all affected versions, if applicable (e.g. 1.13, 2.0, 2.1).
- [ ] Test all commands and procedures, if applicable.
- [ ] Create your PR against `staging`, not `master`. 
- [ ] Provide an estimated date for deploying the doc change. Note: Improvements or fixes can be merged ASAP. 
- [ ] Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects) if you are moving a page.

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing) for more information.

### CODE REVIEW

File: pages/ksphere/konvoy/1.5.0-beta/logging/index.md [hunk 1]
```diff
@@ -171,3 +171,11 @@ The default configuration runs the curator job every day at mid-night and garbag
                 unit: days
                 unit_count: 10
 ```
+
+## Related information
+
+For information on related topics or procedures, refer to the following:
+
+- [resource Recommendations](./recommendations)
```
Comment (juliangieseke) [inline pages/ksphere/konvoy/1.5.0-beta/logging/index.md:179]:
```suggestion
- [Resource Recommendations](./recommendations)
```

File: pages/ksphere/konvoy/1.5.0-beta/tutorials/disable-audit-logs/index.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
+---
+layout: layout.pug
+navigationTitle: Disable Audit Log Collection
+title: Disable Audit Log Collection
+menuWeight: 11
+excerpt: Audit Logs are collected via the FluentBit Addon and enabled for collection by default. In some cases this may not be necessary and turning off audit-log-collection would be ideal. Here we show the procedure to disable Audit Log Collection.
+enterprise: false
+---
+
+### Disable Audit Log Collection
```
Comment (samvantran) [inline pages/ksphere/konvoy/1.5.0-beta/tutorials/disable-audit-logs/index.md:10]: I vote to put the contents of this page in `fluentbit/index.md` as it's closest to the addon info. Right now tutorials are not ordered or categorized so you have kind of a mish-mash of random tutorials. And I think I remember docs team suggesting tutorials are (or can be a long) instructional guide to completing a set of tasks. A 1 step tutorial for switching an addon component on off seems a bit light.

**Comment by juliangieseke** [comment]
@samvantran uh, I thought there is a two way sync in place 🙈  is there a good way "fixing" this pr w/o redoing it?

**Comment by samvantran** [comment]
@juliangieseke uni-directional! But no worries, I ported it over for you https://github.com/mesosphere/konvoy/pull/1686 please remember to make doc changes in konvoy first in the future 🙏

**Comment by cmastr** [comment]
closing as merged via Konvoy repo.

**Pull request closed without merging**